src/components/BrowserApp.tsx

```
@@ -104,55 +99,47 @@ export default function BrowserApp({ onOpenApp }: BrowserAppProps) {
+  const cdpReady = status?.browser?.cdpReady ?? false;
+  // A launch is in flight, or a process exists that has not bound its port
+  // yet. Both are "something is happening, wait", and both want the faster
+  // poll and the pill over the screen.
+  const starting = actionLoading === "open-browser" || (browserRunning && !cdpReady);
+
   useEffect(() => {
-    fetchStatus();
-    pollRef.current = setInterval(fetchStatus, 5000);
-    return () => { if (pollRef.current) clearInterval(pollRef.current); };
-  }, [fetchStatus]);
+    void fetchStatus();
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/BrowserApp.tsx:110:10
  108 |
  109 |   useEffect(() => {
> 110 |     void fetchStatus();
      |          ^^^^^^^^^^^ Avoid calling setState() directly within an effect
  111 |     const id = setInterval(() => { void fetchStatus(); }, starting ? POLL_BUSY_MS : POLL_IDLE_MS);
  112 |     return () => clearInterval(id);
  113 |   }, [fetchStatus, starting]);

(react-hooks/set-state-in-effect)